src/components/CategoryNavContainer/CategoryNav/CategoryNav.js

```diff
@@ -83,6 +88,7 @@ CategoryNav.propTypes = {
     id: PropTypes.string.isRequired,
   })).isRequired,
   cityId: PropTypes.string.isRequired,
+  cityName: PropTypes.string.isRequired,
```

Comment:
This throws a propTypes error in the console on all cities/overview. I thought it was always passed in. So you could either put ` || STRINGS.ALL_CITIES` somewhere further up the tree or revert to not required since you have `const cityName = this.props.cityName || STRINGS.ALL_CITIES;` in this file already.

Although the preferred approach for logic like `const cityName = this.props.cityName || STRINGS.ALL_CITIES;` is to use defaultProps and keep the component cleaner. That's a minor thing though.